src/hooks/useVenuePoll.ts

```diff
@@ -19,10 +19,10 @@ export const useVenuePoll = () => {
   const { userId } = useUser();
 
   const voteInPoll = useCallback(
-    (pollVote: PollVoteBase) => {
+    async (pollVote: PollVoteBase) => {
       if (!venueId) return;
 
-      voteInVenuePoll({ pollVote, venueId });
+      await voteInVenuePoll({ pollVote, venueId });
     },
     [venueId]
   );
```

Comment:
Why not use `useAsyncFn` here rather than in `src/components/molecules/ChatPoll/ChatPoll.tsx`? It makes more sense to wrap it up/abstract it in the hook wouldn't it?

Reply:
@0xdevalias well, I've checked `useAsyncFn` usage in the app 
and every time it's used directly in the component where the `isLoading` state is required. 

And if I wrap the`useVenuePoll` hook, I ought to update hook imports and set `isLoading` as a prop for `<ChatPoll/>`. It seems to me this approach (wrap the hook) could increase the complexity. WDYT?

Reply:
@yurii-lubynets you can still return isPollVoteLoading from the `useVenuePoll` and use it there. It makes sense to me

Reply:
discussed internally width @mike-lvov, fixed in 1d3b5688b0f98e34f710a6d2cb8fb2ca296afcb7